### DISCUSSION TITLE
JS8call

### DISCUSSION THREAD

**Discussion by jasonmtravis** [gdyuldin/x6100_gui#23] [Ideas]
https://bitbucket.org/widefido/js8call/src/js8call/

With GUI keyboard, and/or external keyboard support

**Comment by gdyuldin** [comment]
I never used JS8call. As far as I understand - a keyboard is required. Also - I'm not sure, that such small screen as X6100 has will be suitable for such communication.
Perhaps, the last might be solved after adding a webserver to the radio and use tablet as a external display.

**Reply by jasonmtravis** [reply]
JS8 is designed as a long-form variant of FT8/FT4. People use CW on the x6100 for long-form contacts and contesting, this is the same idea. 
The webserver sounds workable as long as the it was actively decoding the traffic and accepting input in a readable space. Can the x6100 be configured with a hotspot SSID so that a tablet can connect direct over wireless? That would be great for mobile.

Thanks for the consideration and your efforts Georgy. You and Oleg continue to be a boon to the community.

73,
Jason - W5JMT

**Comment by sa5qed** [comment]
JS8Call would of course be VERY NICE, but I guess it's pretty feature rich and it will be quite a challenge both from a resource/processing point of view as well as UI wise.
But please have a go at it, Georgy, but expect we will have to carry a PC for for running JS8Call  for a while.
You have already done magic with this radio!!!
73 de SA5QED

**Comment by DeepRadio01** [comment]
Would a limited version of JS8 call be possible?

**Comment by gdyuldin** [comment]
Maybe. I still have no time to take a look on it.

**Reply by FosteringCrow** [reply]
> Maybe. I still have no time to take a look on it.

I'd like to second the JS8call feature request. The project has been taken over by a new team, who would likely be happy to help implement JS8call within your fantastic x6100 custom FW .

Even a simple decode-only js8call window to start could be SO GREAT, then maybe a simple text input option like FT8/FT4. A lot of JS8call's features can be used with manual formatting within the a text input box by the end user, so it could be relatively simple to have a bare-bones implementation have 99% of the features within the full progrma.

Someone also ported JS8call android recently. I don't know how that helps you, now you know hahah.

Finally, thank you for such awesome work! I really appreciate how great this project is :D

I hope you have a great evening!

**Reply by ki9ng** [reply]
https://github.com/ki9ng/x6100-js8-engine this is my js8 send only engine for the x6100. I use it to send aprs messages over the js8 network to aprs gateways.

**Comment by ki9ng** [comment]
I have implemented JS8 POTA spotting capability. I would appreciate testers and feedback. https://github.com/ki9ng/x6100_gui/releases/tag/potaspotter-v1.0.0
I plan to implement some further js8 to aprs messaging features... SMS and email

**Reply by FosteringCrow** [reply]
If it can send regular JS8 massages as well I can test it this Thursday on my weekly net and send out messages and check for ACKs via a second radio or via x6100 feeding audio to a computer running stock JS8call.

I don't think there was much, if any, encoder/decoder   change between JS8call and js8-improved.

Also thank you for making such an awesome tool!!
